metpetdb_api/settings/staging.py

```diff
@@ -46,7 +46,7 @@
     'rest_framework',
     'rest_framework.authtoken',
     'django_extensions',
-    'vendor.djoser.djoser',
```

Comment:
@luciencd note that I had included djoser in this repository because I needed a slight modification: https://github.com/kristallizer/djoser/commit/0e3b5d62d88cec8c6d949785dce487753d8edf59

You might have to check if that is still needed or not.